### DISCUSSION TITLE
Feature Request: Colour Options

### DISCUSSION THREAD

**Discussion by andrewfairlie** [craftcms/cms#8796] [Ideas]
### Description
It would be great if a future version of Craft allows for users to tweak their control panel preferences to have different colour schemes and light/dark mode. Kind of like how Mac OS does it.

![image](https://user-images.githubusercontent.com/1772888/76541369-623ad800-647b-11ea-96b6-2f9ab867ca38.png)

**Comment by JayBox325** [comment]
Very keen on an admin darkmode.

A really nice option would be to initially check OS preferences, but then also have a toggle to hard set your preference.

**Reply by bengreenbank** [reply]
Much like the Craft documentation site!
